main/modes/flight/3denv.h

```diff
@@ -0,0 +1,1537 @@
+#include <stdint.h>
+const uint16_t model3d[] = {
```

Comment:
Not a fan of assets, eh? I don't love variables in headers without `#ifndef` guards, but I suppose it won't compile if multiple symbols with the same name are detected.

Reply:
It's that it's a big buffer, and no reason for it not to be in flash instead of RAM.  I need RAM for the intermediate geometry stage.

Reply:
This was one of the reasons I wish I could have mmap'd assets, but this almost feels better?

Reply:
I can, however, guard the .h